okapi-core/src/main/java/org/folio/okapi/logging/FolioLoggingContext.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.okapi.logging;
+
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+import org.apache.logging.log4j.core.LogEvent;
+import org.apache.logging.log4j.core.config.plugins.Plugin;
+import org.apache.logging.log4j.core.lookup.StrLookup;
+
+/**
+ * This class should be used for storing context variables
+ * and use them in logging events.
+ */
+@Plugin(name = "FolioLoggingContext", category = StrLookup.CATEGORY)
+public class FolioLoggingContext implements StrLookup {
```

Comment:
"Folio" can be removed from the class name because "folio" is already in the package name: "org.folio.okapi.logging" or "org.folio.okapi.common"

Reply:
In log4j2.properties the name "FolioLoggingContext" used as: $${FolioLoggingContext:userid}. If it is just $${LoggingContext:userid}, the name is not self-explaining and there can be some collisions with some other _LoggingContext_

Reply:
Having 
`
@Plugin(name = "FolioLoggingContext", category = StrLookup.CATEGORY)
public class LoggingContext implements StrLookup {
`
for me also looks not so good.